adoption.html

```diff
@@ -7,8 +7,8 @@
 ---
 
 <div id="maingrid" class="ui container {{ page.id }}">
-{%- assign pages = site.data.sections | group_by:"page" | sort:"page" | reverse -%}
-{% for tracked_page in pages %} 
+{%- assign pages = site.data.sections | group_by:"page" | sort:"page" | rev -%}
```

Comment:
so your sorting order doesnt work for me and i get Adult, donations and home. But I wanted Home at the top, same for the data.json since the reverse ordering gives us the same ordering as the navbar has. so I put it back to reverse for now.